lib/diagnostic.rb

```diff
@@ -108,10 +108,12 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = 'person = {:favorite_number => 42, :first_name => "joe"}
+'
 
 # Set a default return value to your `person` hash that concatenates the value
 # of person[:first_name] and a message about them.
 # Write your response as a Ruby string.
 
-Diagnostic.hash_default = 'your answer here'
+Diagnostic.hash_default = 'person.new { |hash, key| hash[key] = "#{person.first_name}" like the number #{avorite_number}"}
```

Comment:
So this is using the new constructor, we were looking for the short hand version:

`'person.default = "#{person[:first_name]} is great"'`